### PR TITLE
Update advanced-spam-filtering-asf-options.md

### PR DESCRIPTION
**Pull request by andypunt**
Updating article to show deprecation

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/advanced-spam-filtering-asf-options.md [hunk 1]
```diff
@@ -18,15 +18,15 @@ description: "Advanced spam filtering  options give administrators the ability t
 
 # Advanced spam filtering options
 
+>[!NOTE]
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/advanced-spam-filtering-asf-options.md]:
```suggestion
> [!NOTE]
```

File: microsoft-365/security/office-365-security/advanced-spam-filtering-asf-options.md [hunk 2]
```diff
@@ -18,15 +18,15 @@ description: "Advanced spam filtering  options give administrators the ability t
...
+> The Anti-spam policy Advanced Spam filter settings are being deprecated at the time of this writing. Our recommended settings for these are to turn them **OFF**. The functions that were available in the Advanced Spam filter is being incorporated into the other parts of the filtering stack.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/advanced-spam-filtering-asf-options.md]:
```suggestion
> The Advanced Spam filter settings of the Anti-spam policy are currently being deprecated. Our recommended settings for these are to turn them **OFF**. The functions that were available in the Advanced Spam filter is being incorporated into other parts of the filtering stack.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mypil** [comment]
@MSFTTracyP  - PR has been copy edited and is ready for final review. 

Thank you.

cc: @kenwith

**Pull request merged**